charts/navi-back/values.yaml

```
@@ -646,6 +648,25 @@ routesharing:
       security.protocol: PLAINTEXT
     fileProperties: {}
 
+# @section Traffic lights processing. Leave with defaults, FOR FUTURE RELEASE
+# @param trafficLights.enabled If traffic lights processing enabled
+# @param trafficLights.projects List of projects, for which traffic lights are processed
+# @param trafficLights.topic Topic to use for traffic lights processing
+# @extra trafficLights.kafka.properties Properties as supported by librdkafka, see `kafka` section and comments
+# @skip trafficLights.kafka.properties.bootstrap.servers
+# @skip trafficLights.kafka.properties.security.protocol
+# @param trafficLights.kafka.fileProperties Properties stored in file, see `kafka` section and comments
+
+trafficLights:
+  enabled: false
+  projects: []
+  topic: traffic-lights-topic
```

нужно сделать обязательным параметром
```suggestion
  topic: ''
```
топик относится к кафке, лучше его там задавать:
```
trafficLights:
  kafka:
    properties:
      bootstrap.servers: ''
      security.protocol: PLAINTEXT
    fileProperties: {}
    topic: ''
```

---

перенёс, сделал пустыми по умолчанию и required